### PR TITLE
hero, about us and new burger menu design.

### PR DESCRIPTION
**Pull request by YAQ0T**
do new style for burger menu as yousef state.
and finish the "hero panel" and "about us" section for PC not for mobile.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,171 +52,155 @@
             </label>
           </div>
           <ul class="nav">
-            <li><a href="./pages/solution.html">solutions <i class="mobile fas fa-chevron-right"></i></a></li>
-            <li id="products"><a href="./pages/products.html">products <i class="mobile fas fa-chevron-right"></i></a>
+            <li>
```
Comment (yousefomari) [inline index.html:55]: arrange your code 
دائما خليه مرتب

File: index.html [hunk 2]
```diff
@@ -51,171 +52,155 @@
...
                 <div class="container">
+                  <label class="sup-menu-header mobile" for="sup-menu-check"><i class="fa-solid fa-arrow-left"></i>
+                    products</label>
                   <div class="menu-items-lvl1 sup-menu">
                     <ul>
+
                       <li id="boilers">
+                        <input type="checkbox" name="sup-menu-lvl2-check" id="sup-menu-lvl2-check" class="hidden-input"
+                          aria-hidden="true">
                         <a href="#">
```
Comment (yousefomari) [inline index.html]: fix the url
if it's not direct to anywhere try javascript:void(0)

File: style/about-us.css [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  top: -110px;
+  right: 0;
+  width: 24%;
+  max-width: 300px;
+  background-color: var(--primary-color-brand-03);
+  color: var(--text-inverse);
+}
+.who-we-are article p {
+  padding: 30px 25px;
+  margin: 0;
```
Comment (yousefomari) [inline style/about-us.css]: each section in single file

File: style/about-us.css [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  font-size: 1.35rem;
+  font-weight: 300;
+}
+.who-we-are article button {
+  background-color: transparent;
+  color: var(--text-inverse);
+  padding: 0px 25px 30px;
```
Comment (yousefomari) [inline style/about-us.css]: 0 without px

File: style/header.css [hunk 5]
```diff
@@ -86,7 +86,7 @@
   border-color: var(--primary-color-brand-01);
 }
 
-#burger-menu-check {
+.hidden-input {
```
Comment (yousefomari) [inline style/header.css]: this should be a general class, not belong to the header

File: index.html (hunk 2 above)
Comment (YAQ0T) [reply index.html]: ok great thanks

File: style/header.css [hunk 6]
```diff
@@ -228,10 +210,30 @@
 .menu-items li {
   padding: 10px 20px 10px;
 }
-#boilers:hover .sup-menu.menu-items-lvl2,
-#firetube:hover .sup-menu.menu-items-lvl3 {
-  visibility: visible;
-  opacity: 1;
+
```
Comment (yousefomari) [inline style/header.css]: please split the menu CSS code into separate file "navigation.css"

File: style/about-us.css (hunk 3 above)
Comment (YAQ0T) [reply style/about-us.css]: ok don't worry this is my bad, I though they were one section. thanks for the comment

Comment (YAQ0T) [reply style/about-us.css]: Done
Screenshot 2024-05-28 113749

File: style/about-us.css (hunk 4 above)
Comment (YAQ0T) [reply style/about-us.css]: Done

File: style/header.css (hunk 5 above)
Comment (YAQ0T) [reply style/header.css]: Done

File: style/header.css (hunk 6 above)
Comment (YAQ0T) [reply style/header.css]: Done, but I have included the main menu in navigation.css, if I miss understand it I'm willing to change it.
image

File: index.html (hunk 2 above)
Comment (YAQ0T) [reply index.html]: Done

**Pull request merged**